### PR TITLE
Documentation: Scenarios 

### LINKED ISSUE #1346: Update Use Case documentation with quickstart screenshots/instructions
### Add a new page 🆕

**What documentation you'd like us to add?**
Add "Custom" use case page. Add screenshots and descriptions for "scenarios"

**Where in the documentation tree?**
Usage

### Edit existing content 📝

**Which page(s) do you want us to edit?**
"Usage"

**What do you think should be changed?**
Rename "Usage"/"Use Cases" to "Scenarios"

- [ ] Update the documentation (0.25d) - @shreyamalviya 
  - [x] Update scenarios link in Landing Page

### PR DESCRIPTION
**Pull request by shreyamalviya**
# What does this PR do? 

Fixes #1346  

## PR Checklist
* [x] Have you added an explanation of what your changes do and why you'd like to include them?
* [x] Is the TravisCI build passing? 
* [x] Was the CHANGELOG.md updated to reflect the changes?
* [x] Was the documentation framework updated to reflect the changes?

## Testing Checklist

* [ ] ~Added relevant unit tests?~
* [x] Have you successfully tested your changes locally? Elaborate:
    > Tested by running Hugo 
* [x] If applicable, add screenshots or log transcripts of the feature working

image
---
image
---
image
---
image
---

### CODE REVIEW

File: docs/content/usage/scenarios/_index.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+title = "Scenarios"
+date = 2020-08-12T12:52:59+03:00
+weight = 3
+chapter = true
+pre = "<i class='fas fa-map-marked-alt'></i> "
++++
+
+# Scenarios
+
+This section describes <i>Scenarios</i> in the Infection Monkey.
```
Comment (mssalvatore) [inline docs/content/usage/scenarios/_index.md:11]:
```suggestion
This section describes the different attack <i>Scenarios</i> that the Infection Monkey can simulate.
```

File: docs/content/usage/scenarios/_index.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+{{% notice note %}}
+Don't worry! The Infection Monkey uses safe exploiters and does not cause any permanent system modifications that could impact security or operations.
+{{% /notice %}}
+
+The Infection Monkey has pre-existing scenarios to simulate common types of attacks that take place. These scenarios, when selected, manipulate the default configuration values and only show the configuration fields relevant to the selected scenario. This makes it possible for you to quickly run the Monkey on your network with a targeted configuration.
```
Comment (mssalvatore) [inline docs/content/usage/scenarios/_index.md:17]:
```suggestion
The Infection Monkey has pre-built scenarios to simulate common types of attacks that take place. These scenarios, when selected, manipulate the default configuration values and only show the configuration fields relevant to the selected scenario. This makes it possible for you to quickly run the Monkey on your network in order to accomplish a specific objective.
```

**Review by mssalvatore — CHANGES_REQUESTED** [review]

**Review by ilija-lazoroski — APPROVED** [review]

File: docs/content/usage/scenarios/_index.md [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+pre = "<i class='fas fa-map-marked-alt'></i> "
++++
+
+# Scenarios
+
+This section describes the different attack <i>Scenarios</i> that the Infection Monkey can simulate.
+
+{{% notice note %}}
+Don't worry! The Infection Monkey uses safe exploiters and does not cause any permanent system modifications that could impact security or operations.
+{{% /notice %}}
```
Comment (VakarisZ) [inline docs/content/usage/scenarios/_index.md:15]: Why do we need this notice here? What does the safety of exploiters have to do with exploiters? There might be scenarios that don't even use the exploiters

File: docs/content/usage/scenarios/_index.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+The Infection Monkey has pre-built scenarios to simulate common types of attacks that take place. These scenarios, when selected, manipulate the default configuration values and only show the configuration fields relevant to the selected scenario. This makes it possible for you to quickly run the Monkey on your network in order to accomplish a specific objective.
```
Comment (VakarisZ) [inline docs/content/usage/scenarios/_index.md:17]: The technical details of what happens when a user clicks on a scenario are not relevant, they won't help the user to achieve his goals. The purpose of this page is to help the user make a choice of scenario. I would maybe reword to:
"Choose a scenario that best describes your use case. Choosing a "Custom" scenario will allow you to fine-tune your simulation and access some features not available in other scenarios. [Read more about configuring a custom simulation](link-to-custom-page). "

File: docs/content/usage/scenarios/_index.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+![Choose scenario](/images/usage/scenarios/choose-scenario.png "Choose a scenario")
+
+To exit a scenario and select another one, click on "Start Over".
+
+![Start over](/images/usage/scenarios/start-over.png "Start over")
+
+You can also refer to [our FAQ](../../faq) for more specific questions and answers.
```
Comment (VakarisZ) [inline docs/content/usage/scenarios/_index.md:25]: Does FAQ contain something explicit to scenarios? If yes, then link to that specific bulltepoint. If not, no need to mention it here

File: docs/content/usage/scenarios/custom-scenario/_index.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+date: 2021-07-28T14:36:02+05:30
+description: "Configure a custom scenario to test your network's defenses."
+weight: 100
+pre: "<i class='fas fa-edit'></i>"
+chapter: true
+---
+
+# Custom
+
+If none of the existing scenarios fit your use case, you can choose the "Custom" scenario and configure the simulation according to your needs.
```
Comment (VakarisZ) [inline docs/content/usage/scenarios/custom-scenario/_index.md:12]: I'd say that choosing custom is not only for cases when "none of the existing scenarios fit your use case", but also for cases when you want additional functionality. I would reword to something like:
"The Infection Monkey is a versatile breach and attack simulation tool. Choose a "Custom" scenario to access all of its capabilities. This will allow you to enhance, optimize and fine-tune the simulation."

**Review by VakarisZ — APPROVED** [review]
Good job, but see the suggestions

File: docs/content/usage/scenarios/_index.md (hunk 3 above)
Comment (shreyamalviya) [reply docs/content/usage/scenarios/_index.md:15]: This was there previously on the "Use cases" page. I agree that it seems a little out of place here but I'd rather keep it since it is a vital piece of information. Do you have any suggestions on where else to move it?

File: docs/content/usage/scenarios/_index.md (hunk 5 above)
Comment (shreyamalviya) [reply docs/content/usage/scenarios/_index.md:25]: Removed

File: docs/content/usage/scenarios/_index.md (hunk 4 above)
Comment (shreyamalviya) [reply docs/content/usage/scenarios/_index.md]: WDYT?
image
